packages/common/src/global.css

```diff
@@ -1,6 +1,6 @@
 html,
 body {
-  font-family: 'Roboto', sans-serif;
+  font-family: 'Inter', sans-serif;
```

Comment:
This might break some styling in other places, we definitely need to test all pages to make sure that everything still works with this font.

Alternatively, I think we should do this in a separate PR. Then we are not bringing this PR out of scope.

Reply:
removed, it was a request from danny